zetaclient/tss/keygen.go

```diff
@@ -122,16 +122,28 @@ func (k *keygenCeremony) iteration(ctx context.Context) (shouldRetry bool, err e
 	case k.blockThrottled(zetaHeight):
 		return true, nil
 	case zetaHeight < keygenHeight:
+		// connected peers only includes the connections established to other peers
+		connectedPeers := len(k.tss.GetP2PHost().Network().Conns())
+		keygenMembers := len(keygenTask.GranteePubkeys)
+
 		k.logger.Info().
 			Int64("keygen_height", keygenHeight).
 			Int64("zeta_height", zetaHeight).
-			Msg("waiting for keygen block to arrive or new keygen block to be set")
+			Int("connections_established_to_peers", connectedPeers).
+			Int("total_members_for_keygen", keygenMembers).
```

Comment:
How these fields have been chosen? Do we use them other places? They look a bit verbose to me for field name.

Checking as we are trying to make the fields consistent throughout the logging system